frontend/src/page-generation/sermon-pages.js

```diff
@@ -0,0 +1,115 @@
+const hash = require("crypto-js/sha256");
```

Comment:
it seems discontinued and advise to use node crypto. https://www.npmjs.com/package/crypto-js

Reply:
Yes, but we want to still be able to support older node servers. Especially if we end up migrating to a custom server to serve gatsby, we'll need to support node <= v18

Plus thisnis just being used for a simple hash algorithm. There's no cybersecurity involved in what we're doing here, hence no vulnerabilities.